frontend/packages/component-library/package.json

```diff
@@ -19,12 +19,155 @@
   "type": "module",
   "exports": {
     "./package.json": "./package.json",
+    "./alert": {
```

Comment:
You'll need to rebase the code, the require statements will now use the extension `.js` and the imports will use `mjs`.

Reply:
I'll wait until your pr https://github.com/code-dot-org/code-dot-org/pull/63447 will be merged and then will do. Thank you very much for updating everything! 🤩